examples/i18n/ru/lib/calculator.rb

```diff
@@ -11,9 +11,11 @@ def result
     @stack.last
   end
 
+  # rubocop:disable Lint/BinaryOperatorWithIdenticalOperands
```

Comment:
Instead of doing in-line stuff can we use the config file and comment why we're doing this.

- If we know why and we won't fix it, document why
- If we're not sure, put it into the todo file so we can tackle it later

Reply:
When I use in-line rubocop comment, that apply to a single line in the code. Is this possible with the rubocop config file?

Some comments may be missing.

Reply:
I realise I missed this.

The reason we use the yml file (And potentially allow the exclude across the whole file), is it documents the behaviours in one place, so I know where to check.

Using the magic comments makes tracking down and debugging / optimising code a real pain point